src/people.rs

```diff
@@ -213,6 +224,22 @@ impl ContextPeopleExt for Context {
         initialized_value
     }
 
+    fn initialize_person_property<T: PersonProperty + 'static>(
```

Comment:
```suggestion
    /// Sets an initial value for a property, overriding the initialize() function if one is defined.
    /// Does *not* fire an event. Panics if a value is already set.
    fn initialize_person_property<T: PersonProperty + 'static>(
```

Reply:
Well I'm not sure it overrides it. It just bypasses it.